meddlr/metrics/ssfd.py

```diff
@@ -0,0 +1,260 @@
+from typing import Sequence
+
+import torch
+from torch import nn
+
+from meddlr.metrics.functional.image import mse
+from meddlr.metrics.metric import Metric
+from meddlr.ops import complex as cplx
+from meddlr.utils import env
+
+# Pre-trained model for computing SSFD. This is a UNet model trained on a self-supervsed
+# in-painting pretext task on the fastMRI knee dataset.
+SSFD_HUGGINGFACE_URL = "https://huggingface.co/philadamson93/SSFD/resolve/main/model.ckpt"
+
+
+class SSFD(Metric):
```

Comment:
There is a lot of shared structure between the LPIPS and SSFD metrics. consider adding a TODO to refactor these classes to extract shared logic into a parent class (e.g. `class PerceptualMetric`)